src/main/java/com/tenx/ms/retail/order/service/OrderService.java

```diff
@@ -0,0 +1,68 @@
+package com.tenx.ms.retail.order.service;
+
+import com.tenx.ms.retail.order.domain.OrderEntity;
+import com.tenx.ms.retail.order.domain.OrderItemEntity;
+import com.tenx.ms.retail.order.rest.dto.constant.OrderStatusEnum;
+import com.tenx.ms.retail.product.domain.ProductEntity;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+import com.tenx.ms.retail.order.repository.OrderRepository;
+import com.tenx.ms.retail.product.repository.ProductRepository;
+import com.tenx.ms.retail.store.repository.StoreRepository;
+import com.tenx.ms.retail.order.rest.dto.Order;
+import com.tenx.ms.retail.order.rest.dto.OrderItem;
+import com.tenx.ms.retail.order.rest.dto.OrderResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderService {
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Autowired
+    private StoreRepository storeRepository;
+
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Transactional
```

Comment:
You can use Transactional annotation when you have multiple update/insert operations to the database in one single method for "rollback all" purpose. However, I see that you have only one save action here, so this annotation is not required.